cluster/images/kubemark/BUILD

```diff
@@ -0,0 +1,34 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build", "docker_push")
+
+docker_build(
+    name = "image",
+    base = "@debian//image",
```

Comment:
I don't think kubemark needs debian anymore; we might be able to get away with busybox or scratch now.